poi-ooxml/src/main/java/org/apache/poi/xslf/usermodel/XSLFTextRun.java

```diff
@@ -157,6 +160,52 @@ private static void fetchFontColor(CTTextCharacterProperties props, Consumer<Pai
         }
     }
 
+    @Override
```

Comment:
can you add javadoc on public API methods? include `@since POI 5.2.4`

Reply:
Done